.github/workflows/ci.yml

```diff
@@ -1,27 +1,66 @@
-name: CI workflow
-on: [push, pull_request]
+name: CI
+
+on:
+  push:
+    paths-ignore:
+      - 'docs/**'
+      - '*.md'
+  pull_request:
+    paths-ignore:
+      - 'docs/**'
+      - '*.md'
+
 jobs:
   test:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+
     strategy:
       matrix:
         node-version: [10.x, 12.x, 14.x]
+        os: [macOS-latest, windows-latest, ubuntu-latest]
+
     steps:
-    - uses: actions/checkout@v2
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: ${{ matrix.node-version }}
-    - name: Install Dependencies
-      run: npm install --ignore-scripts
-    - name: Test
-      run: npm test
+      - uses: actions/checkout@v2
+
+      - name: Use Node.js
+        uses: actions/setup-node@v2.1.5
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Install Dependencies
+        run: |
+          npm install --ignore-scripts
+
+      - name: Check Licenses
+        run: |
+          npm run license-checker --if-present
```

Comment:
Either add the license checker or remote the step

Reply:
Removed with https://github.com/fastify/fastify-accepts-serializer/pull/37/commits/f923bfce10cca4d75dedb458914bf667e52b6cc1